iolib.go

```diff
@@ -278,8 +278,13 @@ func fileCloseAux(L *LState, file *lFile) int {
 		L.Push(LTrue)
 		return 1
 	case lFileProcess:
-		if file.stdout != nil {
-			file.stdout.Close() // ignore errors
+		// When stdout is piped, close it
```

Comment:
I didn't really want to make a change to where this is in this PR, but shouldn't the stdin close go _AFTER_ the `wait` ?

Reply:
Ok - I really do think it should go after the wait; moving it.

Reply:
Hmm… maybe not… afterall, you have to read all you care about beforehand… it's more likely that the stdout pipe would already be closed… if, for instance, you had done `f:read("*a")` you'd already have waited to completion and the close of the file would only be to capture the statusCode.